lstchain/visualization/plot_drs4.py

```diff
@@ -0,0 +1,88 @@
+
+from matplotlib import pyplot as plt
+
+import numpy as np
+from matplotlib.backends.backend_pdf import PdfPages
+from ctapipe.io import event_source
+from lstchain.calib.camera.r0 import LSTR0Corrections
+
+
+__all__ = ['plot_drs4',
+           ]
+
+channel = ['HG', 'LG']
+
+
+def plot_waveforms(data_file, pedestal_file, run= 0 , plot_file="none"):
+    """
+     plot camera calibration quantities
+
+     Parameters
+     ----------
+     data_file:   pedestal run
+
+     pedestal_file:   file with drs4 corrections
+
+     run: run number of data to be corrected
+
+     plot_file:  name of output pdf file
+     """
+
+    # plot open pdf
+    if plot_file != "none":
+        pp = PdfPages(plot_file)
+
+    plt.rc('font', size=15)
+    offset_value=400
+    tel_id = 1
+
+    # r0 calibrator
+    r0_calib = LSTR0Corrections(pedestal_path=pedestal_file, offset=offset_value,
+                                r1_sample_start=2, r1_sample_end=38, tel_id=tel_id )
+
+    # event_reader
+    reader = event_source(data_file, max_events=8)
+
+    pix = 0
+    pad = 420
```

Comment:
What is `pad`?

Reply:
pad is the pad number where I want to locate the plot